### PR TITLE
[PR] Introduce horizontal navigation, update CSS and JS to go through Grunt

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: functions.php [hunk 1]
```diff
@@ -42,3 +53,228 @@ function nara_enqueue_scripts() {
...
+			wp_enqueue_script( 'wsu-cycle', get_template_directory_uri() . '/js/cycle2/jquery.cycle2.min.js', array( 'jquery' ), spine_get_script_version(), true );
+			wp_enqueue_style( 'genericons', get_template_directory_uri() . '/styles/genericons/genericons.css', array(), spine_get_script_version() );
+		}
+	}
+
+	// Enqueue scripting for the entire parent theme.
+	wp_enqueue_script( 'wsu-spine-theme-js', get_template_directory_uri() . '/js/spine-theme.js', array( 'jquery' ), spine_get_script_version(), true );
+
+	// Dequeue TablePress stylesheet.
+	wp_dequeue_style( 'tablepress-default' );
```
Comment (philcable) [inline functions.php:186]: Probably drop this through line 192 (well, definitely 188 through 192).

File: src/css/02-site-header-and-nav.css [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+	border-color: #981e32;
+	width: 12em;
+}
+
+.site-search li.search {
+	margin-top: -.25em;
+}
+
+.site-search input.search-submit {
+	background: url(./images/ico-search-thin.svg);
```
Comment (philcable) [inline src/css/02-site-header-and-nav.css:38]: This reference will need to be updated.

File: src/css/02-site-header-and-nav.css [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+	.spine-animating.spine-move-left #glue {
+		transform: translate(-100vw);
+	}
+
+	.spine-mobile #spine footer {
+		width: calc(100vw - 54px);
+	}
+
+	.spine-mobile #wsu-signature {
+		background-image: url(./images/esfcom-logo-horiz.svg);
```
Comment (philcable) [inline src/css/02-site-header-and-nav.css:127]: This reference will also need to be updated.

File: src/css/02-site-header-and-nav.css [hunk 4]
```diff
@@ -0,0 +1,544 @@
...
+
+	#jacket #binder #spine header {
+		background-color: transparent;
+		height: 68px;
+		margin: 2.5em 0 2.2em;
+		position: static;
+	}
+
+	#wsu-signature {
+		background-image: url(./images/esfcom-logo.svg);
```
Comment (philcable) [inline src/css/02-site-header-and-nav.css:219]: This one, too!

File: src/css/99-misc-style.css [hunk 5]
```diff
@@ -0,0 +1,633 @@
...
+/**** People and organizations ****/
+
+/******************************
+  Homepage
+******************************/
+.home-banner {
+
+	/* Permalink - use to edit and share this gradient:
+	   http://colorzilla.com/gradient-editor/#8dd1db+0,a7d6db+47,dadfd9+100 */
+	background: url(assets/top_clouds.png) top center, url(assets/clouds.png) top center, url(assets/bottom-dirt.png) bottom center, #8dd1db; /* Old browsers */
```
Comment (philcable) [inline src/css/99-misc-style.css:548]: These paths _might_ need updated too.

**Review by philcable — APPROVED** [review]
Looks good! I made a few comments in the style sheets and one in `functions.php` - all super easy stuff, so I'm going to merge now and let @charlesjdillon update those in the next PR.

**Pull request merged**